projects/plugins/boost/app/assets/src/js/features/lcp/status/error-details.tsx

```diff
@@ -1,10 +1,86 @@
 import FoldingElement from '$features/critical-css/folding-element/folding-element';
 import { recordBoostEvent } from '$lib/utils/analytics';
-import { Notice } from '@automattic/jetpack-components';
+import { getRedirectUrl, Notice } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
 import { __, _n, sprintf } from '@wordpress/i18n';
 import { useLcpState } from '../lib/stores/lcp-state';
+import { LcpErrorDetails } from '../lib/stores/lcp-state-types';
 import styles from './error-details.module.scss';
 
+type PageErrorProps = {
+	url: string;
+	error: LcpErrorDetails;
+};
+
+const PageError = ( { url, error }: PageErrorProps ) => {
+	const getErrorLabel = ( { type, meta }: LcpErrorDetails ) => {
+		if ( type === 'http-error' && meta?.code !== undefined ) {
+			return createInterpolateElement(
+				sprintf(
+					/* translators: %d is the HTTP Status Code */
+					__(
+						'Boost received HTTP error <b>%d</b> when attempting to analyze this page. Please check that the page is publicly accessible, and try again.',
+						'jetpack-boost'
+					),
+					meta.code
+				),
+				{
+					b: <strong />,
+				}
+			);
+		}
```

Comment:
Let's only check the type here.

The reason for this being - if something happens and there's no `meta.code`, an incomplete http error would show the generic one.

But if it's an http error with an unknown code, that at least gives us some idea that it's at least related to the loading of the page.

I've also updated the messaging.

```suggestion
		if ( type === 'http-error' ) {
			return createInterpolateElement(
				sprintf(
					/* translators: %d is the HTTP Status Code */
					__(
						'Boost encountered an HTTP error <b>%d</b> when attempting to analyze this page. Please make sure it loads correctly in incognito mode, and try again.',
						'jetpack-boost'
					),
					meta?.code ?? 'unknown'
				),
				{
					b: <strong />,
				}
			);
		}
```